src/Microsoft.ML.Data/Training/TrainContext.cs

```diff
@@ -403,7 +372,7 @@ internal MulticlassClassificationTrainers(MulticlassClassificationContext ctx)
         /// the top-K accuracy, that is, the accuracy assuming we consider an example with the correct class within
         /// the top-K values as being stored "correctly."</param>
         /// <returns>The evaluation results for these calibrated outputs.</returns>
-        public MultiClassClassifierEvaluator.Result Evaluate(IDataView data, string label, string score = DefaultColumnNames.Score,
+        public MultiClassClassifierEvaluator.Result Evaluate(IDataView data, string label = DefaultColumnNames.Label, string score = DefaultColumnNames.Score,
```

Comment:
>string label = DefaultColumnNames.Label [](start = 77, length = 39)

would assigning defaults here be error prone, this allows the users to trip and provide a name for the label column in the estimators, and leave it to the default name here? We have not assigned defaults in the estimators.

Reply:
Actually, all the trainer extensions also have default names for `Label` and `Features`. This is why I made this change as well.

---
In reply to: [224950306](https://github.com/dotnet/machinelearning/pull/1252#discussion_r224950306) [](ancestors = 224950306)